### DISCUSSION TITLE
城市智慧監控📹

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#119] [General]
**新聞內容:**
Canon這次與加拿大上市公司VSBLTY Group、RADARAPP合作，整合英特爾VPU開發的AI智慧監視攝影系統，所有監視攝影硬體設備包括鏡頭均由Canon提供，軟體開發由VSBLTY及RADARAPP負責。
監視攝影機已安裝約上千台在墨西哥城Benito Juarez區域，預計2022年逐漸放量至墨西哥其他城市。墨西哥城透過這套AI智慧模組，可串聯城市監控系統，從點到面無死角全面監控，除了能鎖定嫌疑犯降低犯罪率，也能辨識車牌及車流，達成交通管制目的。Canon也順利將系統導入馬來西亞。
Canon認為，智慧監控系統是邁向智慧城市的第一步。透過高速網路布建及AI智慧系統發展，遍布城市監控攝影機將從單鏡頭、單功能的事後追蹤犯罪紀錄，當作事先偵知天眼，以及多功能模組，從執法、交通及環境監控等，透過大數據分析提供管理部門決策使用。

**新聞心得:**
使用城市監控系統來降低犯罪率及協助交通管制，的確能有高效率的解決問題、迅速辨識偵測目標，但若是系統的監控範圍觸犯到人民隱私可能就會造成很多困擾，也許政府對於擺放位置考量需要花一點心思！

**新聞來源:**
https://www.moneydj.com//kmdj/news/newsviewer.aspx?a=9e010cd4-29ec-41e0-be5b-aa969eede4b5

**Comment by pocper1** [comment]
全面監控，能鎖定嫌疑犯降低犯罪率，反之，也能威脅到人民的生活隱私。做到全面監控的同時，監控與隱私的比例要拿捏好~

**Comment by Timmy110403540** [comment]
就像一般的監視器一樣，要擺設在適當的位置，造福大眾。

**Comment by minniewei** [comment]
感覺要看地區耶，如果我家周遭都是這種AI智慧監視攝影系統會讓我想搬走......，有點可怕

**Reply by pocper1** [reply]
有可能攝影機鏡頭還會照到家裡，這樣在家都沒隱私了

**Comment by ghost** [comment]
我覺得如何在打擊犯罪和保護隱私中取得平衡是最大的問題

**Comment by Shiba010** [comment]
會不會未來像psycho pass那樣的系統將在未來出現在真實世界中....
